demo/demo-cse-v2/provider/src/main/resources/application.yml

```diff
@@ -31,6 +31,14 @@ servicecomb:
       address: ${CSE_V2_SC}
       instance:
         watch: false
+    service-description:
+      properties:
+        tag:
+          component: ServiceComponent
+    instance-description:
```

Comment:
```
    service-description:
      properties:
          component: ServiceComponent
```

Is this OK? do not add a new special key for properties, because properties is key-value